recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -0,0 +1,23 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+LICENSE = "CLOSED"
+
+UBOOT_CONFIG[doma] = "xenguest_arm64_defconfig"
+UBOOT_CONFIG_prepend = "doma "
+
+SRCREV = "${AUTOREV}"
+SRC_URI = "\
+    git://github.com/arminn/u-boot.git;protocol=https;branch=mr_rcar_pvblock; \
```

Comment:
Do we really want to be on that merge request (mr_) branch?

Reply:
replaced at xen-troops project

Reply:
I would expect it point to the correct branch right from the start, e.g. in this patch